src/frontend/demo/src/index.tsx

```diff
@@ -22,12 +27,17 @@ async function render() {
     translatedMessages = {};
   }
 
+  // Create the controller
+  const controller = new DefaultController({ url: 'http://localhost:3000' });
```

Comment:
this url will have to be a variable?
```suggestion
  const controller = new DefaultController({ url: 'http://localhost:3000' });
```

Reply:
Yes, it will

Reply:
Just note that right now, the whole demo frontend is just a sandbox to demonstrate and try the lib, so I think until we connect the backend, it can wait.